.changeset/date-dropdown-open-active.md

```
@@ -0,0 +1,5 @@
+---
+'@siemens/ix': patch
+---
+
+Keep the Active look on the `ix-date-dropdown` trigger while the picker is open.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)